docs/api/Chip/Chip.md

```diff
@@ -0,0 +1,24 @@
+Chip
+====
+
+Chips represent complex entities in small blocks, such as a contact.
+
+```jsx
+<Chip avatar={<Avatar>} label="Label text" />
+```
+
+Props
+-----
+
+
+| Name | Type | Default | Description |
+|:-----|:-----|:--------|:------------|
+| avatar | node |  | Avatar element. |
+| className | string |  | CSS `className` of the root element. |
+| deleteIconClassName | string |  | CSS `className` of the delete icon element. |
+| label | string |  | Label |
+| labelClassName | string |  | CSS `className` of the label. |
+| onClick | function |  | Callback function fired when the `Chip` element is clicked.<br><br>**Signature:**<br>`function(event: object) => void`<br>*event:* TouchTap event targeting the element. |
+| onRequestDelete | function |  | Callback function fired when the delete icon is clicked. If set, the delete icon will be shown.<br><br>**Signature:**<br>`function(event: object) => void`<br>*event:* `touchTap` event targeting the element. |
+
+Other properties (not documented) are applied to the root element.
```

Comment:
We might need to rebase on `next` as I think we have improved that wording thanks to you 👍